db/migrate/20201211160500_add_creator_user_to_notes.rb

```diff
@@ -1,6 +1,7 @@
 class AddCreatorUserToNotes < ActiveRecord::Migration[6.0]
```

Comment:
It may be problematic to change a migration that is already rolled out in production (because it would be rolled back with a `change` method different to the method it was migrated with). But I never liked this broken migration so let's try out what happens :D